src/test/scala/ExampleWorkflowSpec.scala

```diff
@@ -103,10 +103,9 @@ class ExampleWorkflowSpec extends AnyWordSpec with Matchers with BeforeAndAfterA
     deleteTestFiles()
   }
 
-  override def afterAll() {
+  override def afterAll(): Unit =
     clearDatabaseDataDir()
-    deleteTestFiles()
-  }
+  deleteTestFiles()
```

Comment:
Seems we are missing some curly braces around this function. I think without the curly braces only `clearDatabaseDataDir()` will be called by `afterAll()`